src/muse/outputs/mca.py

```diff
@@ -243,8 +219,6 @@ def sector_capacity(sector: AbstractSector) -> pd.DataFrame:
 
     capacity = pd.concat([u for u in capa_sector])
     capacity = capacity[capacity.capacity != 0]
-
-    capacity = capacity.reset_index()
```

Comment:
Why don't we need to reset index in this one?

Reply:
Because doing so creates a column called "index" in the output file. I guess `reset_index(drop=True)` would also work

Reply:
It isn't needed here anyway. The other outputs need it because they have a multi-index with "month", "day" and "hour", so `reset_index` is required to create the respective columns, whereas here it's just an integer index.